src/packages/core/collection/collection-view.manager.ts

```diff
@@ -68,7 +68,7 @@ export class UmbCollectionViewManager extends UmbBaseController {
 	#createRoutes(views: ManifestCollectionView[] | null) {
 		let routes: Array<UmbRoute> = [];
 
-		if (views) {
+		if (views && views.length > 0) {
```

Comment:
Consider simplifying this check:
```suggestion
		if (views?.length) {
```